splash/lua_modules/wraputils.lua

```diff
@@ -109,6 +109,31 @@ local function sets_callback(func, storage)
   end
 end
 
+local function pack_callback_return_value(func)
+  return function(cb, ...)
+    local mcb = cb
+
+    if type(cb) == 'function' then
+      mcb = function(...)
+        return { cb(...) }
+      end
+    end
+
+    return func(mcb, ...)
+  end
+end
```

Comment:
The reason is why we pack/inpuck in Lua is that if try to return several values from the passed function (callback) it will be converted to the table, because Python cannot know whether it got several values or a table. Then it passed back to Lua [here](https://github.com/mike1808/splash/blob/014b6f7c0d63ee24eede20ca7eb9289adab9a928/splash/browser_tab.py#L654)